docs/hdf_converter.rst

```diff
@@ -0,0 +1,35 @@
+.. highlight:: bash
+
+.. role:: bash(code)
+   :language: bash
+
+HDF file
+========
+
+Saves data in HDF binary file format. For every input detector a single file with :bash:`.h5` extension is written.
```

Comment:
Maybe just mention an example program which uses this format.

Reply:
It is done below, sample script reading HDF format is provided.